colpali_engine/utils/processing_utils.py

```diff
@@ -13,6 +13,20 @@ class BaseVisualRetrieverProcessor(ABC):
     Base class for visual retriever processors.
     """
 
+    @abstractmethod
+    def get_n_patches(
```

Comment:
shouldn't you implment it for idefics2 as well ?
Even if it's just a raised error to say it's nt supported (cuz of the perceiver).
but at least we have the abs class implemented

Reply:
Ah nice catch, forgot about this ahah